games/bt/post_install.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+CONFIG_PATH=$(sudo -u gameuser $GAMEDIR/${GAME}server details | grep "Config file:" | grep -o "/.*")
+
+# Set server name (Displayed name)
+if [ -z "$GAME_SERVER_NAME" ]; then
+    read -p "Server name: " -e -i $(hostname) GAME_SERVER_NAME || GAME_SERVER_NAME='EcoTurnkeyGSM'
+    echo
+fi
+run_as_user "sed -i 's/name=.*/name=\"${GAME_SERVER_NAME}\"/' \"$CONFIG_PATH\""
+
+# Set server Password
+if [ -z "$GAME_SERVER_PASS" ]; then
+    read -p "Server password: " GAME_SERVER_PASS || GAME_SERVER_PASS=''
+    echo
+fi
+run_as_user "sed -i 's/password=.*/password=\"${GAME_SERVER_PASS}\"/' \"$CONFIG_PATH\""
+
+# Set server Public
+if [ -z "$GAME_SERVER_PUBLIC" ]; then
+    until [[ "$GAME_SERVER_PUBLIC" == "true" || "$GAME_SERVER_PUBLIC" == "false" ]]; do
+        read -p "Set server public (true|false): " -e -i 'true' GAME_SERVER_PUBLIC
```

Comment:
Infinite loop for non-interactive installation, provide default value instead
(multiple occasions)